addons/im_livechat/static/src/embed/external/boot.js

```diff
@@ -39,6 +39,7 @@ import { session } from "@web/session";
         },
         { force: true }
     );
+    registry.category("services").remove("error");
```

Comment:
Maybe a comment on the line so we can remember why we did it without having to blame the file?